source/templates/layouts/default.hbs

```diff
@@ -37,6 +37,16 @@
   {{body}}
 
   <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Huh! I swore it always went at the end. Don't we drop it there in nearly all of our projects?